File: blink.c
#include <stdio.h>
#include <wiringPi.h>
  
#define LED1 14
#define LED2 15

int main(){
    if(wiringPiSetupGpio() < 0){
	printf("Unable to setup wiring Pi");
	return 1;
    }

    pinMode(LED1, OUTPUT);
    pinMode(LED2, OUTPUT);

    for(;;){
	digitalWrite(LED1, 1);
	delay(1000);
	digitalWrite(LED1, 0);
	delay(1000);
	digitalWrite(LED2, 1);
	delay(1000);
	digitalWrite(LED2, 0);
	delay(1000);
    }
    return 0;
}

File: blink.py
import RPi.GPIO as GPIO 
import time

led1 = 7
led2 = 29
led3 = 8
led4 = 10

led = [led1, led2, led3, led4]

GPIO.setmode(GPIO.BOARD)
for i in led:
    GPIO.setup(i, GPIO.OUT)

for k in range(5):
    for j in led:
        GPIO.output(j,GPIO.HIGH)
        time.sleep(0.5)
        GPIO.output(j,GPIO.LOW)
        time.sleep(0.5)

GPIO.cleanup()
